.goreleaser.yml

```diff
@@ -40,18 +40,23 @@ scoops:
       email: kartikayjainwal@gmail.com
       
 aurs:
-  - name: obs-bin
+  - name: obsidian-cli-bin
     homepage: https://github.com/Yakitrak/obsidian-cli
     description: Interact with Obsidian in the terminal. Open, search, create, update and move notes!
     maintainers:
       - 'Kartikay Jainwal <kartikayjainwal at gmail dot com>'
       - 'Jonathan Neidel <aur at jneidel dot com>'
     license: 'MIT'
     private_key: '{{ .Env.AUR_KEY }}'
-    git_url: 'ssh://aur@aur.archlinux.org/obs-bin.git'
+    git_url: 'ssh://aur@aur.archlinux.org/obsidian-cli-bin.git'
     package: |-
-      install -Dm755 "./obs" "${pkgdir}/usr/bin/obs"
-      install -Dm644 "./LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
```

Comment:
License removed on purpose?

Reply:
Yeah, comment on the package suggested it 🙂

Reply:
I tested the completions locally, works fine